src/KubeLibrary/KubeLibrary.py

```diff
@@ -41,46 +41,68 @@ class KubeLibrary(object):
     | Library           KubeLibrary          None    True
 
     """
-    def __init__(self, kube_config=None, context=None, incluster=False, cert_validation=True):
+    def __init__(self, kube_config=None, context=None, api_url=None, bearer_token=None,
+                 ca_cert=None, incluster=False, cert_validation=True):
         """KubeLibrary can be configured with several optional arguments.
         - ``kube_config``:
           Path pointing to kubeconfig of target Kubernetes cluster.
         - ``context``:
           Active context. If None current_context from kubeconfig is used.
+        - ``api_url``:
+          K8s API url, used for bearer token authenticaiton.
+        - ``bearer_token``:
+          Bearer token, used for bearer token authenticaiton.
+        - ``ca_cert``:
+          CA certificate file path, used for bearer token authenticaiton.
```

Comment:
Why the `ca-cert` parameter is required when passing the kubernetes cluster bearer token? 
Would `api_url` and `bearer_token` be all we need to authenticate with the Kubernetes cluster?

Reply:
Yes, correct. Fixed.